scan/src/reusable/InfoHL.re

```diff
@@ -1,8 +1,12 @@
+type theme_t =
+  | THEME_1
+  | THEME_2;
+
 type t =
   | Height(int)
   | Count(int)
   | Float(float)
-  | Text(string)
+  | Text(string, theme_t)
```

Comment:
I think we should make another variant called `Description`. The theme approach makes `Text` complicated.

Reply:
Moreover, please fix the conflict.